src/Core/Resources/Resources.cpp

```diff
@@ -28,6 +31,7 @@ std::string getBaseDir() {
 
 ResourcesLocator::ResourcesLocator( void* symbol, const std::string& offset ) : m_basePath{""} {
     auto libraryPath = cpplocate::getLibraryPath( symbol );
+    std::replace(libraryPath.begin(), libraryPath.end(), '\\', '/');
```

Comment:
maybe use an helper function to have something like
auto libraryPath = getLibraryPath( getRadiumResourcesDir);
in both function

Reply:
I agree with @dlyr

Reply:
This locator code does not work when running on the builtree from XCode. 
see commit fcb1de2a130e01ae1834a67db7b7965a2ca17049 on PR #476 to fix it (also remove duplicated code)